symphony/lib/toolkit/class.sectionmanager.php

```diff
@@ -34,6 +34,11 @@ class SectionManager
      */
     public static function add(array $settings)
     {
+        $defaults = array();
+        $defaults['creation_date'] = $defaults['modification_date'] = DateTimeObj::get('Y-m-d H:i:s');
+        $defaults['creation_date_gmt'] = $defaults['modification_date_gmt'] = DateTimeObj::getGMT('Y-m-d H:i:s');
+        $defaults['author_id'] = '1';
```

Comment:
Because it's a copy of this line https://github.com/symphonycms/symphony-2/blob/2.7.x/symphony/lib/toolkit/class.entry.php#L105

Should we change both ?